docs/auth.md

```diff
@@ -146,9 +147,84 @@ the format for the secret changes we will need to create a separate secret for
 just the service catalog today OR we need to add a new `UserServiceAdapter` that
 understands that secret.
 
+### Bearer Auth
+The below section will focus on the bearer token auth.
+
+#### Configuration
+By default, if no authentication is specified the broker will use bearer token auth. The bearer token authentication will use delegated auth from the [kubernetes apiserver](https://github.com/kubernetes/apiserver) library.
+
+The configuration is to grant access, through [kubernetes RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) roles and role-bindings, to the url prefix. The broker has added a configuration option `cluster_url` to specify the url_prefx. This value will default to `ansible-service-broker`. 
```

Comment:
I'd wrap the line better but the important one is 👍 
TYPO: url_prefx -> url_prefix